deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -231,24 +235,28 @@ private void ingestChildren(ContentContainerObject destObj, Resource parentResc)
 	 * @throws DepositException
 	 * @throws IOException 
 	 */
-	private void ingestFileObject(ContentObject parent, Resource parentResc, Resource childResc)
+	private FileObject ingestFileObject(ContentObject parent, Resource parentResc, Resource childResc)
 			throws DepositException, IOException {
 
 		if (skipResumed(childResc)) {
-			return;
+			return null;
 		}
 
 		// TODO add ACLs
 		WorkObject work = (WorkObject) parent;
 		FileObject obj = addFileToWork(work, childResc);
-		addDescription(work);
-		// add premis events to the file object
+		// Add ingestion event for file object
+		addIngestionEventForChild(obj);
 		addPremisEvents(obj);
+		// add MODS
+		addDescription(work);
```

Comment:
I can't recall why this needed to be moved, but either way I think the description should be getting added to the fileObject rather than the work here (since it is not ingestFileObjectAsWork)